plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -575,6 +631,40 @@
     node: sabrewulf
     protection: yes
 
+- name: Create new VM using Cloud-Init with a username and password
```

Comment:
Why are you writing cloud-init everywhere above, and now Cloud-Init (different capitalization)?

Reply:
Copy-Paste i guess ;)